test/LANDRegistryTest.sol

```diff
@@ -3,6 +3,10 @@ pragma solidity ^0.4.18;
 import '../contracts/land/LANDRegistry.sol';
 
 contract LANDRegistryTest is LANDRegistry {
+  function safeTransferFromToEstate(address from, address to, uint256 assetId, uint256 estateId) external {
```

Comment:
I think we should need this method at LANDRegistry and not only for testing purpose. 

We must avoid the normal transfer to `EstateRegistry` contract cause it can be stolen with a simple script

```
it.only('can be stolen by anyone', async function() {
      // Create Estate with 5 lands owned by user
      const estateId = await createUserEstateWithNumberedTokens()
      // Create another Estate with land (0, 100) owned by anotherUser
      await land.assignMultipleParcels([0], [100], anotherUser, sentByCreator)
      const anotherEstate = await createEstate([0], [100], anotherUser, sentByAnotherUser)
      // Transfer out of Estate the land (0, 2)
      await transferOut(estateId, 2)
      // Transfer from user to EstateRegistry the land (0, 2)
      await unsafeTransferIn(2)
      // Ammend Received the land (0,2) to another Estate
      await estate.ammendReceived(anotherEstate, 2, sentByAnotherUser)
      // Another Estate has the land (0,2)
      await assertLandIdAtIndex(anotherEstate, 1, 2)
    })
```

To keep the `transfer` to `EstateRegistry` we should need to store the `oldOwner` of the land received at `onERC721Received` in a new mapping like:

`mapping(string => uint256[]) public oldOwnerLands`

```
function onERC721Received(
    address oldOwner,
    uint256 tokenId,
    bytes estateTokenIdBytes
  )
    external
    onlyRegistry
    returns (bytes4)
  {
    oldOwnerLands[oldOwner].push(tokenId);
    uint256 estateId = _bytesToUint(estateTokenIdBytes);
    _pushLandId(estateId, tokenId);
    return bytes4(0xf0b9e5ba);
  }
```

And on `ammendReceived` do some kind of validation with it